src/politodown/polito.py

```diff
@@ -70,9 +70,26 @@ async def get_videostores(year: int) -> dict[str, Videostore]:
                 continue
 
             videolesson_name = videolesson.text.strip()
-            cor, = data_regex.search(videolesson["href"]).groups()
-            videolessons[videolesson_name] = \
+            if data_regex.match(videolesson["href"]).group(1):
+                cor = data_regex.search(videolesson["href"]).group(2)
+                videolessons[videolesson_name] = \
                 Videostore(year, videostore_name, videolesson_name, cor)
+            else:
+                inc_regex = re.compile(r"dokeosLez\(\'(\d+)\'\)")
+                inc, = inc_regex.match(videolesson["onclick"]).groups()
+                data = await session.get(
+                    httpx.URL(
+                        urls.did/"pls/portal30/sviluppo.materiale.json_dokeos_par",
+                        params={"inc": inc}
+                    )
+                )
+                data.raise_for_status()
```

Comment:
TODO: Check wether it's safe to raise an error in this part of code, or if it should just return instead.